docs/reference/ImageDraw.rst

```diff
@@ -387,8 +387,9 @@ Methods
                     the number of pixels between lines.
     :param align: If the text is passed on to
                   :py:meth:`~PIL.ImageDraw.ImageDraw.multiline_text`,
-                  ``"left"``, ``"center"`` or ``"right"``. Determines the relative alignment of lines.
-                  Use the ``anchor`` parameter to specify the alignment to ``xy``.
+                  ``"left"``, ``"center"``, ``"right"`` or ``"justify"``. Determines
+                  the relative alignment of lines. Use the ``anchor`` parameter to
+                  specify the alignment to ``xy``.
```

Comment:
Maybe version it something like this?
```suggestion
                  ``"left"``, ``"center"``, ``"right"`` or ``"justify"``. Determines
                  the relative alignment of lines. Use the ``anchor`` parameter to
                  specify the alignment to ``xy``.
                  
                  .. versionadded:: 11.2.0 ``justify``
```
Plus for the others below.

And let's also add to release notes.